src/nav/AppContainer.js

```diff
@@ -66,6 +67,8 @@ class AppContainer extends PureComponent {
       actions.fetchEssentialInitialData();
       actions.fetchRestOfInitialData(pushToken);
       actions.fetchEvents();
+      actions.fetchUsersStatus(); // do a initial fetch
+      setInterval(() => actions.fetchUsersStatus(), config.activePingInterval);
```

Comment:
this might lead to actively pinging presence response with focus `true`, so even when the app is not in focus (the app state is false) , the user presence is updated on server.